examples/ffd/src/app_conf.h

```diff
@@ -123,12 +123,13 @@
 #define appconfAUDIO_PIPELINE_BUFFER_ENABLED    1
 #endif
 
-/* The number of frames (appconfAUDIO_PIPELINE_FRAME_ADVANCE) to store in a ring
- * buffer while in low power mode. This may be tuned to ensure that unvoiced
- * speech that is a pre-cursor to voiced speech in a wake-word such as "he" part
- * of "hello" is captured and relayed to the inference engine. */
-#ifndef appconfAUDIO_PIPELINE_BUFFER_NUM_FRAMES
-#define appconfAUDIO_PIPELINE_BUFFER_NUM_FRAMES 32
+/* The number of packets to store in a ring buffer while in low power mode,
+ * where each packet containing appconfAUDIO_PIPELINE_FRAME_ADVANCE frames.
+ * This may be tuned to ensure that unvoiced speech that is a pre-cursor to
+ * voiced speech in a wake-word such as "he" part of "hello" is captured and
+ * relayed to the inference engine. */
+#ifndef appconfAUDIO_PIPELINE_BUFFER_NUM_PACKETS
+#define appconfAUDIO_PIPELINE_BUFFER_NUM_PACKETS 32
```

Comment:
I am not sure if there is a desired name/term for this, but this variable and other variables in the application were not clearly/correctly named.

Reply:
Each audio frame contains 240 samples, or 15ms of audio.

There have been spots where frames and samples were used interchangeably (incorrectly), but in this case, the buffer enqueues 1 frame of 240 samples, so appconfAUDIO_PIPELINE_BUFFER_NUM_FRAMES would be correct.

To prevent confusion, we should fix `frame_count`, for the audio_pipeline_output handlers, as it has not been frame count for a long time.  `frame_count` is samples per frame, so sample_count would be more appropriate.

Reply:
I think you are confusing `appconfAUDIO_PIPELINE_BUFFER_NUM_FRAMES` with something else. This is a #define I added with the introduction of this ring buffer. Its only purpose is for sizing the ring buffer.

Reply:
Or maybe I am confused by the term frame. I have been thinking of it as equivalent to a sample in the case where there is only one mic.